File: CodeWars_Python/calculate_averages.py
#!/usr/bin/env python
# python 3.4.3

# Write function avg which calaculates average of numbers in given list.


def find_average(array):
    try:
        return sum(array)/len(array)
    except ZeroDivisionError:
        return 0

print('Example test')
array_1 = [1, 2, 3]
print(find_average(array_1))


print('Edge test')
array_2 = []
print(find_average(array_2))

File: CodeWars_JS/calculate_averages.js
// Write function avg which calaculates average of numbers in given list.

function find_average(array) {
  var total = 0;
  for (var i = 0; i < array.length; i++){
    total += array[i];
  }
  return total / array.length;
}

console.log(find_average([1,1,1]));
console.log(find_average([1,2,3]));
